com.unity.render-pipelines.high-definition/Editor/Sky/AtmosphericScattering/FogEditor.cs

```diff
@@ -157,18 +150,16 @@ public override void OnInspectorGUI()
                                 }
                             }
                         }
+                    }
 
-                        PropertyField(m_DirectionalLightsOnly);
-                        PropertyField(m_Anisotropy, s_AnisotropyLabel);
-                        if (m_Anisotropy.value.floatValue != 0.0f)
-                        {
-                            EditorGUILayout.Space();
-                            EditorGUILayout.HelpBox(
-                                "When the value is not 0, the anisotropy effect significantly increases the performance impact of volumetric fog.",
-                                MessageType.Info, wide: true);
-                        }
+                    PropertyField(m_DirectionalLightsOnly);
+                    if (PropertyField(m_Anisotropy, s_AnisotropyLabel) && m_Anisotropy.value.floatValue != 0.0f)
```

Comment:
Same here

Reply:
Yeah, if the Anisotropy is not rendered it will return false and therefore the content won't be rendered